assets/src/scss/blocks/Accordion/AccordionStyle.scss

```diff
@@ -105,8 +105,8 @@
     }
 
     .accordion-btn {
-      margin: $sp-3 10% 0 10%;
-      line-height: 2.5;
+      margin: $sp-3 0 0 0;
+      line-height: var(--font-size-2xl--font-family-primary);
```

Comment:
This tokens refers to `font-size` and you're assigning to `line-height`

Reply:
You're right. I have reverted this change since there is no token for a line-height equal to `2.5rem`